tests/test_renderer.py

```diff
@@ -1,11 +1,14 @@
+import os
 import pytest  # type: ignore
+from xml.dom import minidom  # type: ignore
```

Comment:
Maybe this should go to your test function instead?

Reply:
OK, it is not clear to me why to prefer one over the other

Reply:
> OK, it is not clear to me why to prefer one over the other

Because this is a renderer test, a renderer test is not something that has any knowledge of XML or parsing. But it's just a personal opinion, feel free to disregard.